src/creatures/players/components/player_stash.cpp

```diff
@@ -0,0 +1,154 @@
+/**
+ * Canary - A free and open-source MMORPG server emulator
+ * Copyright (©) 2019-2022 OpenTibiaBR <opentibiabr@outlook.com>
+ * Repository: https://github.com/opentibiabr/canary
+ * License: https://github.com/opentibiabr/canary/blob/main/LICENSE
+ * Contributors: https://github.com/opentibiabr/canary/graphs/contributors
+ * Website: https://docs.opentibiabr.org/
+ */
+
+#include "creatures/players/components/player_stash.hpp"
+
+#include "database/database.hpp"
+#include "items/item.hpp"
+#include "creatures/players/player.hpp"
+#include "game/scheduling/save_manager.hpp"
+
+PlayerStash::PlayerStash(Player &player) :
+	m_player(player) { }
+
+void PlayerStash::add(uint16_t itemId, uint32_t amount) {
+	const auto it = m_stashItems.find(itemId);
+	if (it != m_stashItems.end()) {
```

Comment:
What about using the created find method, since you are not using the iterator.

```suggestion
	if (find(itemId)) {
```

Reply:
I think that is not a big deal, because it will save only one line.